# request
float32[] ee_pos
float32[] curr_joint_angles
---
# response
float32[] joint_angles
